resources/rest-service/cloudify/install_agent.py

```diff
@@ -159,6 +181,17 @@ def install(self):
         finally:
             self.runner.rm_dir(path)
 
+    def _get_certificate_file(self, path=None):
+        if not self.cloudify_agent.get('verify_rest_certificate'):
+            return None
+
+        if self.cloudify_agent.get('local_rest_cert_file'):
```

Comment:
TODO check

i think actually at this point we cna't use local_rest_cert_file yet, and instead we should've kept the previous implementation